libraries/AP_NavEKF2/AP_NavEKF2.cpp

```diff
@@ -554,6 +558,15 @@ const AP_Param::GroupInfo NavEKF2::var_info[] = {
     // @RebootRequired: True
     AP_GROUPINFO("EXTNAV_DELAY", 50, NavEKF2, _extnavDelay_ms, 10),
 
+    // @Param: FLOW_MASK
+    // @DisplayName: Optical flow use bitmask
+    // @Description: Bitmask controlling if the optical flow dara is fused into the main navigation estimator and/or the terrain estimator.
```

Comment:
small typo, "dara" should be "data" in FLOW_MASK param description

Reply:
Will fix